pages/weather/index.tsx

```diff
@@ -4,6 +4,8 @@ export default function WeatherPage() {
     return (
         <div>
             <p>Hot weather</p>
+            <p> New feature</p>
+            <h1>Cold weather</h1>
```

Comment:
Harold this is not going to work.

Cold weather originates from Sweden!

Reply:
What are you talking about? It's 29 degrees right now. (insert raised eyebrow emoji)

Reply:
Okej looks good...